companion/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+	"id": "PPT-VIDEO",
+	"name": "PPT-VIDEO",
+	"shortname": "PPT/VIDEO",
+	"description": "This software aims to use the companion feature to use more than one computer on the network and be able to control the opening of video and ppt simultaneously.",
+	"version": "1.0.0",
```

Comment:
we normally leave this at 0.0.0 and the build process will pick up the version from the package.json